thoth-documentation/src/main/paradox/advanced/multi-command.md

```diff
@@ -0,0 +1,41 @@
+# Multiple command handling in the same transaction
+
+To send multiple commands in the same transaction, transaction context should be instantiated manually and passed to command handler along with commands.
+
+Code below show implementation of multiple command processing to transfer money from one account to another in the same transaction, [see complete code here](https://github.com/MAIF/thoth/blob/master/sample/src/main/java/fr/maif/thoth/sample/api/BankController.java).
+
+```java
+try(Connection connection = dataSource.getConnection()) {
+    connection.setAutoCommit(false);
+    var withdrawResult = eventProcessor
+            .batchProcessCommand(
+                    connection,
+                    List.of(new BankCommand.Withdraw(transferDTO.from, transferDTO.amount))
+            );
+    var depositResult = eventProcessor
+            .batchProcessCommand(
+                    connection,
+                    List.of(new BankCommand.Deposit(transferDTO.to, transferDTO.amount))
+            );
```

Comment:
Processing multiple command like this, should be with two different processors because with the same processor you just have to pass the list of commands in `batchProcessCommand`.